src/FffGcodeWriter.cpp

```diff
@@ -555,6 +563,9 @@ void FffGcodeWriter::processRaft(const SliceDataStorage& storage)
 
     for (int raftSurfaceLayer = 1; raftSurfaceLayer <= train->getSettingAsCount("raft_surface_layers"); raftSurfaceLayer++)
     { // raft surface layers
+        // make sure that we are using the correct extruder to print raft
```

Comment:
You also need to set the extruder for the interface layer.
The raft would then be like so:
raft base, make prime blob for each extruder, raft interface, raft surface layers